ktext.md

```diff
@@ -330,6 +330,8 @@ However ProperTree will handle this for you, so you need not concern yourself
   * Adds support for a large variety of SATA controllers, mainly relevant for laptops which have issues seeing the SATA drive in macOS. We recommend testing without this first.
   * macOS Big Sur Note: [CtlnaAHCIPort](https://github.com/dortania/OpenCore-Install-Guide/blob/master/extra-files/CtlnaAHCIPort.kext.zip) will need to be used instead due to numerous controllers being dropped from the binary itself
     * Catalina and older need not concern
+* [CPUTopology](https://github.com/b00t0x/CpuTopologyRebuild)
```

Comment:
Despite not explicitly required, [CPUTopology](https://github.com/b00t0x/CpuTopologyRebuild) may help Alder Lake users to optimize heterogeneous core configuration (P-Cores and E-Cores)